jwql/utils/maths.py

```diff
@@ -0,0 +1,154 @@
+"""Various math-related functions used by the ``jwql`` instrument monitors.
+
+Authors
+-------
+
+    - Bryan Hilbert
+
+Use
+---
+
+    This module can be imported as such:
+
+    >>> import maths
+    mean_val, stdev_val = maths.mean_stdev(image, sigma_threshold=4)
+ """
+
+import numpy as np
+
+from astropy.modeling import fitting, models
+from astropy.stats import sigma_clip
+from scipy.optimize import curve_fit
+from scipy.stats import sigmaclip
+
+
+def double_gaussian(x, amp1, peak1, sigma1, amp2, peak2, sigma2):
```

Comment:
@bhilbert4 Does this functionality not already exist in something like `scipy`?

Reply:
Nothing built-in for fitting the sum of two Gaussians simultaneously that I found.